src/index.css

```diff
@@ -95,4 +95,26 @@
 .style-title-page{
   font-weight: bold;
   font-size: 30px;
+}
+
+.btn-table{
+  background-color: #dbb82d;
+  border: none;
+  cursor: pointer;
+  font-size: 16px;
+  color: white;
+}
+
+.mr{
```

Comment:
I think bootstrap already has this class `mr`, [Documentation](https://getbootstrap.com/docs/4.0/utilities/spacing/)

Reply:
yes love I know that boostrap has it but the change did not apply to me

Reply:
Really? Lets check it @SaraLopez03